### DISCUSSION TITLE
General Queries

### DISCUSSION THREAD

**Discussion by savidudu** [keepassxreboot/keepassxc#8327] [Q&A] [answered]
Hello All,

I have few general queries:

a) Would Crowdstrike, Tanium etc. agents pickup on the unlock password or any CLI based autofill scripts if that was used ?

b) Is there a way to set minimum database unlock credentials strength to open the database ?

c) Is there an option to **FORCE** minimum password strength ?

d) Auto Close time for the KeepassXC Database ?

Thanks

**Comment by droidmonkey** [comment, marked as answer]
A) If a forensic memory image is made with those tools, then yes... your master key and/or password may be retrievable. Albeit very hard to discern from any other data due to address space layout randomization. So without knowing the password you wouldn't know what to search for. 

B) no

C) no

D) you can deploy a config file with the desired setting, but it is not locked to that setting.

We are considering enterprise enforced settings in a future release.
